circle.yml

```diff
@@ -110,13 +110,13 @@ jobs:
       - deploy:
           name: Publish Java Services To Maven Central
           command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+            if [ "${CIRCLE_BRANCH}" == "pg-android-core" ]; then
```

Comment:
@Guardiola31337 I believe this needs to be changed back to `master` before merging.